components/input/input.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../component.css">
+    <script src="https://kit.fontawesome.com/67165fa09e.js" crossorigin="anonymous"></script>
+    <title>Scrole UI</title>
+</head>
+
+<body>
+    <div class="main-wrapper">
+        <header class="navbar-wrapper">
+            <div class="icon-div">
+                <img class="main-icon" src="../../images/Scroll-UI logo.svg" alt="logo">
+                <span class="icon-text">Scrole</span>UI
+            </div>
+            <nav class="navbar">
+                <ul class="nav-link">
+                    <li><a href="">Home</a></li>
+                    <li><a href="">Documentation</a></li>
+                </ul>
+            </nav>
+            <a href="#">Examples</a>
+        </header>
+        <nav class="side-navbar">
+            <ul class="side-ul">
+                <h1>Components</h1>
+                <li class="side-li"><a href="../avatar/avatar.html">Avatar</a></li>
+                <li class="side-li"><a href="../alert/alert.html">Alert</a></li>
+                <li class="side-li"><a href="../badge/badge.html">Badge</a></li>
+                <li class="side-li"><a href="../button/button.html">Button</a></li>
+                <li class="side-li"><a href="../card/card.html">Cards</a></li>
+                <li class="side-li"><a href="../image/image.html">Image</a></li>
+                <li class="side-li"><a href="../input/input.html">Input</a></li>
+                <li class="side-li"><a href="">Nav 3</a></li>
+                <li class="side-li"><a href="">Nav 1</a></li>
+                <li class="side-li"><a href="">Nav 2</a></li>
+                <li class="side-li"><a href="">Nav 3</a></li>
+                <li class="side-li"><a href="">Nav 3</a></li>
+            </ul>
+        </nav>
+        <section class="main-section">
+            <div class="section-heading">Inputs</div>
+            <div class="section-subhead">Basic and Animated Inputs .</div>
+            <p class="section-para">Use class "input-box" for animated input box .</p>
+            <div class="component-example flex-row flex-align-center flex-justify-center flex-gap1">
+                <div class="component-input">
+                    <input class="input-box" placeholder=" " type="text">
```

Comment:
You should add an id of name to this input, so that the label which has a "for" attr of name is associated with this input. Else the for by itself does nothing.

Reply:
ok would fix that!